hack/whitespace.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+
+function fix() {
+    find . -type f \
+         ! -path './.git/*' \
```

Comment:
a good start for files to skip would be $(git ls-files) [you'll need to add vendor/ but that's understandable]